lib/sidekiq_unique_jobs/lock/until_executing.rb

```diff
@@ -33,6 +33,9 @@ def lock(&block)
       def execute
         callback_safely if locksmith.unlock
         yield
+      rescue StandardError => ex
+        reflect(:execution_failed, item, ex)
+        locksmith.lock(wait: 1)
```

Comment:
Shouldn't the exception be rethrown here?

Reply:
Indeed, sorry! Fixed in #712

Reply:
@axos88 relesed as v7.1.23 I believe